test/fixtures/autofix/jQuerySap.js

```diff
@@ -46,6 +46,8 @@ sap.ui.define(["sap/base/strings/NormalizePolyfill"], async function (NormalizeP
 	// The old API always returns a logger instance. So in case the return value
 	// is accessed, the call should not be migrated:
 	jQuery.sap.log.debug().info();
+	const debug = (msg, logger) => logger ? logger(msg) : jQuery.sap.log.debug(msg);
+	debug("msg 2", jQuery.sap.log.debug("msg 1"));
```

Comment:
I added this case which I think shouldn't be migrated

Reply:
Please take a look here, we have agreed not to migrate those. It's the same as here: https://github.com/SAP/ui5-linter/pull/614/files/3f323b738af724a864a93f19d814723d55898ace..fdcab5292817cf90f8d4454c257205fea0a53162#diff-ecfbd6fee929bed2c5f1bc6dd0aa5b47380c9f64479567968446c3b3fb937926R57

Basically, if there's a comma, it could be within a test and the test could check for `this`

Reply:
Not sure I understand, are you saying this should be migrated or not?

Currently `debug("msg 2", jQuery.sap.log.debug("msg 1"));` becomes `debug("msg 2", Log.debug("msg 1"));`, while `jQuery.sap.log.debug(), jQuery.sap.log.info();` stays the same.

Reply:
Now it's fixed

Reply:
I added another case where `debug("msg 2", (jQuery.sap.log.debug("msg 1")));` (notice the additional brackets) is migrated to `debug("msg 2", (Log.debug("msg 1")));`

Reply:
Handling for return statements and arrow functions with implicit return is also missing.
E.g.
```js
function log() {
	return jQuery.sap.log.error("FOO");
}
doSomething(() => jQuery.sap.log.error("FOO"));
```

Reply:
Now should be better :)
Is there anything else that comes to your mind that those could be used for to return a value (`this`)?